src/okareo/okareo.py

```diff
@@ -131,6 +133,36 @@ def upload_scenario_set(
             print(e.content)
             raise
 
+    def download_scenario_set(
+        self,
+        scenario: Union[ScenarioSetResponse, str],
+        file_path: str = "",
+        project_id: Union[Unset, str] = UNSET,
+    ) -> Any:
+        try:
+            scenario_id = (
+                scenario if isinstance(scenario, str) else scenario.scenario_id
+            )
+            url = f"{BASE_URL}/v0/scenario_sets_download/{scenario_id}"
+            headers = {
+                "accept": "application/json",
+                "api-key": self.api_key,
+            }
+            response = httpx.get(
+                url,
+                headers=headers,
+            )
+            filename = response.headers["content-disposition"].split('"')[1]
+            if file_path != "":
+                filename = file_path
+            binary_file = open(filename, "wb")
+            binary_file.write(response.content)
+            binary_file.close()
+            return binary_file
+        except UnexpectedStatus as e:
```

Comment:
If it's not using client stub generated classes, does it raise UnexpectedStatus?